demo/app/components/app_footer_component.rb

```diff
@@ -3,6 +3,7 @@
 class AppFooterComponent < ViewComponent::Base
   def call
     render CitizensAdviceComponents::Footer.new do |c|
+      c.logo(url: "/")
```

Comment:
I think we've discussed it before but it feels like the logo in the design system docs should lead to the main CA site (or be a design system specific logo)